extensions/src/common/extension/decorators/newBlocks.ts

```diff
@@ -1,15 +1,20 @@
-import { BlockMetadata, Argument, ReturnTypeByBlockType, ScratchBlockType, ToArguments } from "$common/types";
+import { BlockMetadata, Config, Argument, ReturnTypeByBlockType, ScratchBlockType, ToArguments } from "$common/types";
 import { block } from "$common/extension/decorators/blocks";
 import { ExtensionInstance } from "..";
 import { TypedMethodDecorator } from ".";
 import type BlockUtilityWithID from "$scratch-vm/engine/block-utility";
 
+export const scratch = {
+    reporter: makeDecorator("reporter"),
+    command: makeDecorator("command"),
```

Comment:
I propose, instead of having `scratchVersions`, we a `versions` property here, so the decorator can be used as `scratch.versions`